cases/management/commands/enrich_case_overview.py

```
@@ -0,0 +1,2122 @@
+            uploaded.open("rb")
+            raw = uploaded.read()
+        except Exception:
+            return None
+        finally:
+            try:
+                uploaded.close()
+            except Exception:
+                pass
+        try:
+            ole = olefile.OleFileIO(raw)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)